### DISCUSSION TITLE
Newbie has 2 questions about rooting and dating a tree

### DISCUSSION THREAD

**Discussion by bonisko** [iqtree/iqtree2#242] [Q&A] [answered]
Dear all,

I recently installed IQ-tree v 2.3.4 for Windows.  I have executed Iqtree2 with a Clustal alignment of 98 sequences which gives a tree that is very similar to the output of Mega11.

Next I created a calibration file with dates for 3 nodes.  IQtree completed with no errors.  I opened the output file (nwk) which I can root, but the nodes I used to calibrate are REALLY different than what I specified in my DateFile.  (Eg. 2 Gya vs 370 Mya!)

Any idea what I am doing wrong?

In Mega, I must root a tree before dating, but I see no way to do this in IQtree.  Might this be my problem?

Thanks to anyone with an idea!

Bruce
Berkeley CA

**Comment by roblanf** [comment]
Hi @bonisko,

In order to help you, we'll need more information. Can you provide the full command line, input and output files? Also just tagging @tothuhien since this is partly a dating question.

Rob

**Comment by bonisko** [comment]
Thanks!  13 files zipped and attached.
Bruce

iqtree2 -s AtpBNoSpacesClustal.fas -T AUTO --date DATE_FILE.txt

[13Files_for_Rob.zip](https://github.com/user-attachments/files/15959513/13Files_for_Rob.zip)

**Comment by bonisko** [comment]
Oops.  Forgot to tag @tothuhien 

Thanks!

Bruce

**Comment by tothuhien** [comment, marked as answer]
Hi,
Very sorry for the delay! I'm just back from my holiday so was not aware of this. I see in the command line that you haven't specified the date for the tips. I assumed they are all 0, so could you try again by adding option `--date-tip 0` ?
Hien

**Reply by roblanf** [reply]
Edited this from `--date-tips` to `--date-tip` as corrected by @tothuhien, just so I can mark it as the answer.

**Comment by bonisko** [comment]
Hi Hien,

Thanks!  But this gives an error, although the syntax looks correct.  (See http://www.iqtree.org/doc/iqtree-doc.pdf)

![image](https://github.com/iqtree/iqtree2/assets/33467506/01ec696f-7489-431d-a089-da0eaab1ba36)

C:/Invalid "--date-tips" option.  And I am sure I am passing a zero, not the letter O.

Ideas?

Bruce

**Comment by tothuhien** [comment]
Hi, sorry the option is `--date-tip` and not `--date-tips`, could you try again?
Hien

**Comment by bonisko** [comment]
Problem solved!

Thanks!

Bruce
